code/modules/projectiles/guns/ballistic/bows/bow_types.dm

```diff
@@ -45,3 +45,130 @@
 /obj/item/gun/ballistic/bow/divine/with_quiver/Initialize(mapload)
 	. = ..()
 	new /obj/item/storage/bag/quiver/holy(loc)
+
+#define CHARGE_TIME 1.5 SECONDS
+/obj/item/gun/ballistic/bow/compact
+	name = "compact bow"
+	desc = "Hi-Tech compact energy multi-bow. \n Alt-click to transform."
+	icon_state = "compact_bow"
+	base_icon_state = "compact_bow"
+	inhand_icon_state = "compact_bow"
+	worn_icon_state = "compact_bow"
+	accepted_magazine_type = /obj/item/ammo_box/magazine/internal/bow/compact
+	drawn = TRUE
+	slot_flags = null
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 26
```

Comment:
The other stats make sense to me but I just want to know why it's stronger than an armblade in melee

Reply:
to have some opportunity to fight back when the cartridge is being reloaded.
but you're right, the number is too big

Reply:
(IMO) The weapon shouldn't be capable of everything. If it's supremely good at range, being in melee should put you at a disadvantage. Then the player can play around the disadvantage, either by acquiring a proper melee weapon (at 11tc, you can afford an esword as well) or by playing around it (don't go into melee)

Reply:
in this case, I can leave the base damage of the bow at 15